File: 프로그래머스/0/181830. 정사각형으로 만들기/정사각형으로 만들기.py
def solution(arr):
    col = len(arr)
    row = len(arr[0])
    
    if col > row:
        for i in range(col):
            for j in range(col-row):
                arr[i].append(0)
    elif col < row:
        for i in range(row-col):
            arr.append([0 for _ in range(row)])
                
    return arr

File: 프로그래머스/0/181830. 정사각형으로 만들기/정사각형으로 만들기.java
class Solution {
    public int[][] solution(int[][] arr) {
        int cnt = Math.max(arr.length, arr[0].length);
        
        int[][] answer = new int[cnt][cnt];
        
        for(int i = 0; i < arr.length; i++) {
            for(int j = 0; j < arr[0].length; j++) {
                answer[i][j] = arr[i][j];
            }
        }
                
        return answer;
    }
}